### DISCUSSION TITLE
clarification on the loss function

### DISCUSSION THREAD

**Discussion by weilaigit** [deepmodeling/deepmd-kit#2456] [Q&A] [answered]
Hi all,

I will appreciate it if someone could clarify the definition of the loss function in the deepmd-kit paper (https://arxiv.org/pdf/1712.03641.pdf)

![image](https://user-images.githubusercontent.com/11967008/231563564-e98114b9-742a-4a41-ac2a-9efb5eae6bba.png)

1. I am assuming epsilon is related to the cell energy, i.e. not energy per atom?
2. What is _N_ here, the number of atoms in a cell, and _i_ is the atom index?
3. Is it practical to call delta F_{i} as rmse of force as it only refers to one atom, if _i_ is indeed the atom index?  Mathematically it is correct but it is awkward?

**Comment by njzjz** [comment, marked as answer]
Hi,
We've updated the equations in the [new DeePMD-kit paper](https://doi.org/10.48550/arXiv.2304.09409). Although we didn't change anything on the loss function, the definition of these symbols should be more clear in the new paper.
